tensorflow/lite/micro/BUILD

```diff
@@ -231,10 +225,12 @@ cc_library(
         ":micro_compatibility",
         ":micro_log",
         "//tensorflow/lite/c:common",
-        "//tensorflow/lite/core/api",
+        "//tensorflow/lite/core/api:flatbuffer_conversions",
```

Comment:
This library still directly depends on 'lite/core/api:flatbuffer_conversions'. To remove this dependency, we must expose each of the Op parser functions through the bridge. Not sure this is ideal in terms of maintainability. Keeping this open for feedback.

Reply:
Thanks for calling this out. I think what you have is good. I agree that the benefit of having all the individual functions in the bridge is not worth the overhead it will introduce.